Labfiles/01-prompt-engineering/CSharp/CSharp.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
does the new 2.0 library require net8.0? the VMs we use to host this only has net7, so like to keep it at 7 like the other ones. if aoai 2.0 requires net8.0, I'll have to figure out how to proceed